### ISSUE TITLE
Rerun changed cached chunks

### ISSUE THREAD

**Issue by konstunn** [mpastell/Pweave#86]
Will pweave automatically rerun chunks which were changed since last time they were cached? Or one should handle this manually? And, however, as far as I can see there are only two options on what to run (not to run): rerun everything or nothing. Is it so?

**Comment by fgregg** [comment]
@mpastell would you be interested in a PR that added this functionality?

**Comment by mpastell** [comment]
Yes, a PR would be welcome. It’s actually easy to check if a chunk has changed, but it would probably be more useful to somehow handle depencies between chunks.

**Comment by fgregg** [comment]
I agree that checking dependencies would be great. This is not something that knitr or sweave does. Would you accept a PR that just reran changed chunks.
